hack/make.sh

```diff
@@ -81,6 +81,14 @@ if command -v git &> /dev/null && git rev-parse &> /dev/null; then
 	GITCOMMIT=$(git rev-parse --short HEAD)
 	if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
```

Comment:
Yeah, my goal was to make the change with as little impact as possible so I wouldn't break anything.